VideoWeb/VideoWeb/Controllers/ParticipantsController.cs

```diff
@@ -321,5 +331,65 @@ public async Task<IActionResult> GetCurrentParticipantAsync(Guid conferenceId)
                 return StatusCode(e.StatusCode, e.Response);
             }
         }
+
+        [HttpPost("{conferenceId}/joinConference")]
+        [SwaggerOperation(OperationId = "StaffMemberJoinConference")]
+        [ProducesResponseType(typeof(ConferenceDetailsResponse), (int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.NotFound)]
+        [ProducesResponseType((int)HttpStatusCode.BadRequest)]
+        [Authorize(AppRoles.StaffMember)]
+        public async Task<IActionResult> StaffMemberJoinConferenceAsync(Guid conferenceId,
+            StaffMemberJoinConferenceRequest request)
+        {
+            var username = request.Username.ToLower().Trim();
+            
+            try
+            {
+                var originalConference = await _videoApiClient.GetConferenceDetailsByIdAsync(conferenceId);
+
+                if (originalConference.Participants.Any(x=>x.Username==username))
+                {
+                    _logger.LogDebug($"Staff Member {username} is already in this conference {conferenceId}");
+                    return Ok(originalConference);
+                }
+
+                if (!_participantService.CanStaffMemberJoinConference(originalConference))
+                {
+                    _logger.LogWarning("Staff Member only can view hearing within 30 minutes of the Start time and 2 hours after the hearing has closed");
+                    ModelState.AddModelError(nameof(conferenceId), $"Please select a valid conference {nameof(conferenceId)}");
+                    return BadRequest(ModelState);
+                }
+                
+                _logger.LogDebug("Attempting to assign {StaffMember} to conference {conferenceId}", request.Username,
+                    conferenceId);
+
+                var userProfile = await _userApiClient.GetUserByAdUserNameAsync(username);
+                var staffMemberEmail = userProfile.Email;
+                var claimsPrincipalToUserProfileResponseMapper =
+                    _mapperFactory.Get<ClaimsPrincipal, UserProfileResponse>();
+                var staffMemberProfile = claimsPrincipalToUserProfileResponseMapper.Map(User);
+
+                if (staffMemberProfile.Role == Role.StaffMember)
+                {
+                    await _videoApiClient.AddStaffMemberToConferenceAsync(conferenceId, _participantService.InitialiseAddStaffMemberRequest(staffMemberProfile, staffMemberEmail, User));
+                }
+                var updatedConference = await _videoApiClient.GetConferenceDetailsByIdAsync(conferenceId);
```

Comment:
it looks like we are calling video api too many times here, was it not possible to create a new query that checks whether or not the participant is already in the conference and if they aren't add them if their role is staff member? I know you were creating a new query in video api before however that have any logic around that.